components/panel/body-toggle.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * WordPress dependencies
+ */
+import Dashicon from 'components/icon-button';
+
+function PanelBodyToggle( { opened, label, onClick } ) {
+	return (
+		<button className="components-panel__body-toggle" onClick={ onClick }>
```

Comment:
Not that difficult but my thinking here is that this component could be used as standalone. But maybe I'm optimising for unnecessary features.